plugins/ruby/ruby.cpp

```diff
@@ -1023,7 +1023,7 @@ static VALUE rb_dfmemory_set_clear(VALUE self, VALUE set)
 
 
 /* call an arbitrary object virtual method */
-#ifdef WIN32
+#if defined(_WIN32) && !defined(_WIN64)
```

Comment:
Yeah, this one is... tricky. It might compile with this change, but it'll probably break the plugin in mysterious ways on 64-bit Windows. It *might* be possible to move the assembly to a separate file, but even then, it would probably need more changes to work on x64. @jjyg would know best here.